src/ralph/models/edx/problem_interaction/fields/events.py

```diff
@@ -40,13 +40,13 @@ class CorrectMap(BaseModelWithConfig):
         queuestate (json): see QueueStateField.
     """
 
-    answervariable: Union[Literal[None], None, str]
+    answervariable: Union[Literal[None], None, str] = None
```

Comment:
I wonder if setting a default value for this field makes it not required

Reply:
comment transfered to newer PR